### ISSUE TITLE
ME interface autocrafting sound bug

### ISSUE THREAD

**Issue by DualBallers** [GTNewHorizons/NewHorizons#3552]
2.0.4.9

I have an ME Interface setup to autocraft long enr.naq rods for my naq reactor.
![image](https://user-images.githubusercontent.com/30087707/44913288-32126500-ad2d-11e8-8c9f-51342f3b3ef3.png)

The long rods are crafting using the crafting table recipe using a hammer
![image](https://user-images.githubusercontent.com/30087707/44913519-eb713a80-ad2d-11e8-9232-a9ab7b40c32c.png)

Every time the interface requests crafting of this long rod, a hammering sound will appear "chik-chik-chik".
The sound is loudest when I am close to the ME interface, but even when I am in another dimension (and the ME system is chunkloaded) the sound will play!

I don't know if this is connected - but when you I mouse-over some items in the molecular assembler that are crafted with the hammer (e.g. rocket fins), the hammer sound plays on repeat, as if the item is being crafted repeatedly. It is both weird and really annoying.
![image](https://user-images.githubusercontent.com/30087707/44914199-d5fd1000-ad2f-11e8-9fe5-1c3849992d18.png)

**Comment by boubou19** [comment]
afaik it happens only if a gt tool is used in the craft

**Comment by DoomSquirter** [comment]
sound muffler next to ur assembler ftw

**Comment by richardhendricks** [comment]
Try replacing the recipe with a Forge Hammer?

**Comment by DualBallers** [comment]
@DoomSquirter, haha you're right. The muffler next to the molecular assembler helps a bit.

Ya, the forge hammer crafting solves the immediate issue for the long rod, but it is still weird that the ME system triggers the GT tool sounds in these conditions.

**Comment by draknyte1** [comment]
Any shaped craft involving tools plays a sound.. not weird at all..

**Comment by DualBallers** [comment]
It is just weird when the sound from the autocrafting is played "near" the player when in another dimension.  And it is weird that the crafting sound loops repeatedly when you mouse over certain items in the molecular assembler.

**Comment by richardhendricks** [comment]
Probably using a fakePlayer that ties back to you?

**Comment by DualBallers** [comment]
No, just a regular ME system and an interface with a crafting card. Unless my world anchor is the one "transmitting" the crafting sound? - the anchor is placed kinda close to the molecular assembler.
I could try to move the world anchor a bit about and report back if this affects the sound.

**Comment by DualBallers** [comment]
I tried removing the world anchor and moved the interface with crafting card a bit around, but the crafting sound is still played directly "on" the player, no matter how far you are away. 

If there is nothing to do to fix this you can close the issue, as it can be duct-taped by using a sound muffler by the molecular assembler or a forge hammer recipe instead.
